packages/store-ui/README.md

```diff
@@ -4,12 +4,29 @@
 
 [![NPM](https://img.shields.io/npm/v/@vtex/store-ui.svg)](https://www.npmjs.com/package/@vtex/store-ui) [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 
-## Install
+## Installation
 
-```bash
+From the command line in your project directory, run npm install `@vtex/store-ui` or yarn add `@vtex/store-ui`.
+
+```cmd
+npm install @vtex/store-ui
+# or
 yarn add @vtex/store-ui
 ```
 
+For style, you can use our default theme. To install:
+
+```cmd
+npm install @vtex/theme-b2c-tailwind
+# or
+yarn add @vtex/theme-b2c-tailwind
+```
+
+```tsx
+import { Slider } from '@vtex/store-ui'
+import '@vtex/theme-b2c-tailwind/dist/index.css'
+```
```

Comment:
This bit should replace the one below, right?

Reply:
yep, can be